src/mono/mono/mini/helpers.c

```diff
@@ -274,7 +274,7 @@ MONO_RESTORE_WARNING
 	g_free (cmd);
 #endif
 
-	cmd = g_strdup_printf (ARCH_PREFIX DIS_CMD " %s %s", objdump_args, o_file);
+	cmd = g_strdup_printf ("llvm-objdump -d --arch=rv64gc %s %s", objdump_args, o_file);
```

Comment:
Temporary change, revert? I think this helper should provide llvm-objdump support anyway but it can be done in a separate PR. e.g. @kotlarmilos's PR https://github.com/dotnet/runtime/pull/96332 for similar llvm toolchain and binutils support side-by-side.

Reply:
sorry, forgot to revert it